### DISCUSSION TITLE
Installation error when upgrading to GridPath v16

### DISCUSSION THREAD

**Discussion by jhimelic** [blue-marble/gridpath#1065] [Q&A]
Hi GridPath community. 

I downloaded and installed the latest version of GridPath on my windows machine. Unfortunately, I am receiving an error when testing my installation of Gridpath version 16 (see below). 

Here are the steps I have taken:
- download the source code and documentation zip from github
- unzip the files and store them in my specified gridpath folder
- open up a anaconda prompt terminal
- create a new envt: conda create --name gridpath_v16 python==3.11
- activate the new envt: activate gridpath_v16
- navigate to appropriate gridpath directory w/ requirements.txt file and install gridpath v16: pip install-e .a[all]
-  test my installation of gridpath v16: python -m unittest discover tests

Previously, I've installed other versions of GridPath before without issue; this is the first time i am seeing this error msg while testing my installation. I'm not sure if this is a bug in the test file or if my installation is indeed not correct. 

Thank you

![image](https://github.com/blue-marble/gridpath/assets/98625939/ba5380a6-e1f5-4d87-b85e-90f67c81ae03)

**Comment by usernameKR** [comment]
Try running an example  and i think it will solve it solver  , I think this is upgrade bug in grid-path

**Reply by MahavirSingh20** [reply]
thank you. I tried running an small model, still it is showing error
